sphinx/cmd/quickstart.py

```diff
@@ -48,24 +49,43 @@
 
 TERM_ENCODING = getattr(sys.stdin, 'encoding', None)
 
-DEFAULT_VALUE = {
+EXTENSIONS = collections.OrderedDict([
+    ('autodoc', 'automatically insert docstrings from modules'),
+    ('doctest', 'automatically test code snippets in doctest blocks'),
+    ('intersphinx', 'link between Sphinx documentation of different projects'),
+    ('todo', 'write "todo" entries that can be shown or hidden on build'),
+    ('coverage', 'checks for documentation coverage'),
+    ('imgmath', 'include math, rendered as PNG or SVG images'),
+    ('mathjax', 'include math, rendered in the browser by MathJax'),
+    ('ifconfig', 'conditional inclusion of content based on config values'),
+    ('viewcode',
+     'include links to the source code of documented Python objects'),
+    ('githubpages',
+     'create .nojekyll file to publish the document on GitHub pages'),
+])
+
+BUILDERS = collections.OrderedDict([
+    ('htmlhelp',
+     'generate HTML help support files for Microsoft HTML Help Workshop'),
+    ('latex',
+     'generate LaTeX files that can be processed into PDF and other formats'),
+    ('man', 'generate man pages in groff format'),
+    ('texinfo',
+     'generate Texinfo files that can be processed into Info files'),
+    ('epub', 'generate an EPUB file for e-book readers'),
+])
```

Comment:
How did you choice these builders?

Reply:
These are the builders for which there was configuration in the quickstart template. I can add more, but that should be another patch/pull request, I think?